actions/bump-version/action.yaml

```diff
@@ -22,14 +26,19 @@ runs:
   using: "composite"
   steps:
     - name: Set up bump2version
-      run: pipx install bump2version
+      run: |
+        pip install --upgrade bump2version
```

Comment:
is there a reason why you switched from pipx to pip?

Reply:
actually no. its just that i was facing some troubles so i had to remove anything am not familiar with to isolate and locate the issue. But i am also wondering if there is any advantage to go with `pipx` over `pip`